frontend/src/main/pages/StaffIndexPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useBackend } from 'main/utils/useBackend';
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { Button } from 'react-bootstrap';
+import { useCurrentUser, hasRole} from 'main/utils/currentUser';
+import StaffTable from 'main/components/Staff/StaffTable';
+import { useParams } from "react-router-dom";
+
+export default function StaffIndexPage() {
+  let {id} = useParams();
+
+  const { data: currentUser } = useCurrentUser();
+  const createButton = () => {  
+
+      return (
+          <Button
+              variant="primary"
+              href="/staff/create"
```

Comment:
This is not the right url.  There is no notion of creating a staff member outside the context of a specific course.  So the URL should be:

```suggestion
              href=`/staff/${id}/create`
```

And you need at least a temporary placeholder page that says: 
`Placeholder for Staff Creation Page for Course ${id}` until the page is ready.

Reply:
the changes have been made accordingly